src/core/Workbench.tsx

```diff
@@ -1,267 +1,342 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
-import {useCallback, useEffect, useReducer, useState} from "react";
-import AceEditor from 'react-ace';
+import { useCallback, useReducer } from "react";
+import AceEditor from "react-ace";
 
-import '../../assets/styles.scss';
+import "../../assets/styles.scss";
+import "@awsui/global-styles/index.css";
+import { useToggle } from "../common/useToggle";
+import { useMouseUpAnywhere } from "../common/useMouseUpAnywhere";
+import { Topbar } from "../topbar/Topbar";
+import ThemeProvider from "./ThemeProvider";
```

Comment:
There are lots of changes in this file due to Prettier fixing styling issues. I'll add a comment to the parts that are more related to this PR